Model/ResourceModel/Magazine/Collection.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * @package Styla/Connect2
+ * @author Oskar Wolanin <owolanin@divante.co>
+ * @copyright 2018 Divante Sp. z o.o.
+ * @license See LICENSE_DIVANTE.txt for license details.
+ */
+
+namespace Styla\Connect2\Model\ResourceModel\Magazine;
+
+class Collection extends \Magento\Framework\Model\ResourceModel\Db\Collection\AbstractCollection
+{
+    /**
+     * @var string
+     */
+    protected $_idFieldName = 'id';
+
+    /**
+     * @var string
+     */
+    protected $_eventPrefix = 'styla_magazine_collection';
+
+    /**
+     * @var string
+     */
+    protected $_eventObject = 'magazine_collection';
+
+    /**
+     * Define resource model
+     *
+     * @return void
+     */
+    protected function _construct()
+    {
+        $this->_init('Styla\Connect2\Model\Magazine', 'Styla\Connect2\Model\ResourceModel\Magazine');
+    }
+
+    /**
+     * @return void
+     */
+    public function joinStoreCode()
```

Comment:
Gdzie tego używasz?

Reply:
To z tego co widzę, będzie używane później na kolekcji przy gridzie.